### PR TITLE
adding scripts description to kick start the app in README.md

### PR DESCRIPTION
**Pull request by Shubham-1994**
Adding description of scripts like dev, build, build-static, start so that it is easy for anyone to kick start the app.

### CODE REVIEW

**Comment by invalidtoken** [comment]
@Shubham-1994 We have made some changes to the repo please pull the changes and re-push. 
You can see the changes in issue #11. Sorry for the trouble.

**Comment by invalidtoken** [comment]
Please mention the issue this PR is fixing.

**Comment by Shubham-1994** [comment]
This fix is for issue #17

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "start": "next start",
     "build-static": "next build && next export",
     "lint": "eslint --fix .",
-    "format": "prettier --write './**/*.{js,jsx,ts,tsx,scss,md,json}'"
+    "format": "prettier --write \"./**/*.{js,jsx,ts,tsx,scss,md,json}\""
```
Comment (invalidtoken) [inline package.json:11]: Great job fixing #17

File: README.md [hunk 2]
```diff
@@ -132,3 +132,15 @@ export default Button;
...
+
+These scripts refer to the different stages of developing an application:
+
+#### dev - Runs next which starts Next.js in development mode.
+
+#### build - Runs next build which builds the application for production usage.
+
+#### build-static - Runs next build && next export which builds the application and have a static version of app in the 'out' directory.
+
+#### start - Runs next start which starts a Next.js production server.
```
Comment (invalidtoken) [inline README.md:146]: Thanks for adding the commands.

**Pull request merged**